### PR TITLE
style: more consistent usage of the `as` keyword

### PR DESCRIPTION
**Pull request by a-frantz**

### CODE REVIEW

File: workflows/qc/quality-check-standard.wdl [hunk 1]
```diff
@@ -153,29 +153,29 @@ workflow quality_check {
         prefix=post_subsample_prefix + ".QualityScoreDistribution",
         max_retries=max_retries
     }
-    call fqc.fastqc { input:
+    call fastqc_doc.fastqc { input:
```
[the remark this answers is not in the document]
Comment (adthrasher) [reply workflows/qc/quality-check-standard.wdl:156]: We could do something like 
```
call fastqc.run as fastqc
```
Though that adds another task rename. I don't have a clear thought on what is best.

File: workflows/qc/quality-check-standard.wdl [hunk 2]
```diff
@@ -27,18 +27,18 @@
 
 version 1.1
 
-import "../../tools/fastqc.wdl" as fqc
+import "../../tools/fastqc.wdl" as fastqc_doc
```
[the remark this answers is not in the document]
Comment (a-frantz) [reply workflows/qc/quality-check-standard.wdl:30]: I like that idea, using `_tasks` over `_doc`

File: workflows/qc/quality-check-standard.wdl (hunk 1 above)
[the remark this answers is not in the document]
Comment (adthrasher) [reply workflows/qc/quality-check-standard.wdl]: I think the change to `fastqc_tasks` for the import resolves this as well. Now the tasks becomes
```
call fastqc_tasks.fastqc
```
and any usage of the outputs uses the clearer name `fastqc.result`.

[the remark this answers is not in the document]
Comment (a-frantz) [reply workflows/qc/quality-check-standard.wdl]: As we discussed offline, I think any ideal namespacing scheme we could come up with is going to include `from URI import foo (as bar)`. Until WDL adds support for that I think we're just going to have to settle for compromises.

I just don't see a way to resolve this case (WDL doc shares the name of a task) that makes me happy.

File: workflows/reference/calc-gene-lengths.wdl [hunk 3]
```diff
@@ -39,9 +39,12 @@ workflow calc_gene_lengths {
         Int? max_retries
     }
 
-    call util.calc_gene_lengths as calc { input: gtf=gtf, max_retries=max_retries }
+    call util.calc_gene_lengths as calc_gene_lens { input:
```
Comment (adthrasher) [inline workflows/reference/calc-gene-lengths.wdl:42]: Do we need to abbreviate this?

**Review by adthrasher — APPROVED** [review]

File: workflows/reference/calc-gene-lengths.wdl (hunk 3 above)
Comment (a-frantz) [reply workflows/reference/calc-gene-lengths.wdl:42]: Yes, I didn't realize why at first. The encompassing workflow name is `calc_gene_lengths`. A little surprised they're considered the same namespace. Not the biggest fan of this abbreviation, and I'm realizing a better solution might be to expand "calc"->"calculate" in the workflow name. Or in the task name? Either one would get rid of the name collision.

Comment (adthrasher) [reply workflows/reference/calc-gene-lengths.wdl:42]: That said, renaming the task is likely easier.

**Pull request merged**